doc/releases/release-notes-3.2.rst

```diff
@@ -90,6 +90,69 @@ Deprecated in this release
    valid for specific bindings to specify like :dtcompatible:`gpio-leds` and
    :dtcompatible:`fixed-partitions`.
 
+
+* Bluetooth mesh
+
+  * The following functions related to the Bluetooth mesh Configuration Client model have changed:
+
+    * :c:func:`bt_mesh_cfg_comp_data_get()` is deprecated in favor of :c:func:`bt_mesh_cfg_cli_comp_data_get()`
```

Comment:
Will it be OK to shorten this list to something like:
`All the Bluetooth mesh Configuration Client API with prefix "bt_mesh_cfg_*" is deprecated in favor of a new API with prefix "bt_mesh_cfg_cli_*"`?